userprofile/templates/userprofile/login.html

```diff
@@ -1,99 +1,112 @@
+	</div>
+
+	<div id="login-box-row" class="row">
 
-<div class="row">
+		{% if shibboleth_login %}
+			<div class="col-sm-6 col-md-3 login-box">
+				<div class="thumbnail">
+					<div class="caption">
+						{% if shibboleth_title_text %}
+							<h3>
+								{{ shibboleth_title_text|safe }}
+							</h3>
+						{% endif %}
+						{% if shibboleth_body_text %}
+							<p>
+								{{ shibboleth_body_text|safe }}
+							</p>
+						{% endif %}
+						<p>
+							<a class="btn btn-primary btn-block" role="button" href="{% url 'shibboleth-login' %}?next={{ next|iriencode }}">
+								{{ shibboleth_button_text|safe }}
+							</a>
+						</p>
+					</div>
+				</div>
+			</div>
+		{% endif %}
 
-    {% if shibboleth_login %}
-    <div class="col-sm-6 col-md-3">
-        <div class="thumbnail">
-            <div class="caption">
-                {% if shibboleth_title_text %}
-                <h3>
-                	{{ shibboleth_title_text|safe }}
-                </h3>
-                {% endif %}
-                {% if shibboleth_body_text %}
-                <p>
-                	{{ shibboleth_body_text|safe }}
-                </p>
-                {% endif %}
-                <p>
-                	<a class="btn btn-primary btn-block" role="button" href="{% url 'shibboleth-login' %}?next={{ next|iriencode }}">
-                		{{ shibboleth_button_text|safe }}
-                	</a>
-                </p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+		{% if mooc_login %}
+			<div class="col-sm-6 col-md-3 login-box">
+				<div class="thumbnail">
+					<div class="caption">
+						{% if mooc_title_text %}
+							<h3>
+								{{ mooc_title_text|safe }}
+							</h3>
+						{% endif %}
+						{% if mooc_body_text %}
+							<p>
+								{{ mooc_body_text|safe }}
+							</p>
+						{% endif %}
+						<p>
+							<a class="btn btn-danger btn-block" role="button" href="{% url 'social:begin' 'google-oauth2' %}?next={{ next|iriencode }}">{% trans 'Log in using Google' %}</a>
+						</p>
+					</div>
+				</div>
+			</div>
+		{% endif %}
 
-    {% if mooc_login %}
-    <div class="col-sm-6 col-md-3">
-        <div class="thumbnail">
-            <div class="caption">
-                {% if mooc_title_text %}
-                <h3>
-                  {{ mooc_title_text|safe }}
-                </h3>
-                {% endif %}
-                {% if mooc_body_text %}
-                <p>
-                    {{ mooc_body_text|safe }}
-                </p>
-                {% endif %}
-                <p>
-                    <a class="btn btn-danger btn-block" role="button" href="{% url 'social:begin' 'google-oauth2' %}?next={{ next|iriencode }}">{% trans 'Login Using Google' %}</a>
-                </p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+		<div class="col-sm-6 col-md-3 login-box extra-login">
+			<div class="thumbnail">
+				<div class="caption">
+					<h3>
+						{% trans "Local users" %}
```

Comment:
In my opinion the "Local users" is actually a bit obscure, I wouldn't know what is meant by it. This might just be though, and the additional text explains the login-option better.

Reply:
"Local users" certainly isn't perfect since the term unfamiliar to many users (such as myself), however, it's rather good since it's kind of exactly what it is, thus googling the term comes up with results that are the equivalent of this. We couldn't really come up with a better title. (And I don't think "A+ login" is any better, since at least my first thoughts to that are: "Don't the other login options log in to A+?") (If you have better ideas, please share them!) Yeah, the descriptive text is really, really important with an unclear, unfamiliar and jargon-y term

Reply:
Would "Admin users" do? Since in Finnish it is now "Ylläpidon kirjautuminen"? Just an idea :)

Reply:
Admins don't use that. Our best not-best word was "special login for special people". There are very few if no users wich use this login. Basically, if we need to grant someone access to the system and they can't use either of the normal logins.

Reply:
Okay, seems reasonable :)